packages/data-service/src/connect-mongo-client.ts

```diff
@@ -166,17 +189,26 @@ export async function connectMongoClientDataService({
   //
   // If connectionOptions.sshTunnel is not defined, the tunnel
   // will also be undefined.
-  const [tunnel, socks5Options] = await openSshTunnel(
-    connectionOptions.sshTunnel,
-    logger
+  // (?. because this can be false-y in compass-web)
```

Comment:
Concerns me a bit that this requires some knowledge about how compass-web build is configured and is not enforced by anything else, but this knowledge. I wonder if we can make sure that somehow every time anything from this library is used in compas monorepo 🤔 Is there a way for us to maybe extend this package types with a shared d.ts file in some way that would make every export optional and apply it everywhere through our shared tsconfig? Not a blocker, but maybe if it's low effort to try something like this, it might be worth it?

Reply:
Yeah, fair enough. I think it's a bit cleaner to use the `localPolyfill()` variant in this case, it's not quite as "automatic" like making the types for a given package all-optional, but I don't think the latter can be implemented all _that_ easily either

Reply:
(done in fa14cefbc58fca9fc68fc7a6dcc6f3ba5c24380b)

Reply:
Yep, makes sense, even better probably, very obvious what the expectation is there now 👍